### PR TITLE
add support for 'listed' parameter

### PR DESCRIPTION
**Pull request by examknow**

### CODE REVIEW

File: dronebl/__init__.py [hunk 1]
```diff
@@ -38,11 +38,12 @@ def _post(self,
         return response.read()
 
     def lookup(self,
-            query: Union[str, int],
-            type:  Optional[int]=None,
-            limit: Optional[int]=None
+            query:  Union[str, int],
+            type:   Optional[int]=None,
+            limit:  Optional[int]=None,
+            listed: Optional[int]=2
```
Comment (jesopo) [inline dronebl/__init__.py]: where did 2 come from? is that in a doc somewhere?

Comment (examknow) [reply dronebl/__init__.py]: listed="0|1|2" - from https://dronebl.org/docs/rpc2

0 is not listed, 1 is listed, so 2 must be any (it is also the default if the param is not given)

Comment (jesopo) [reply dronebl/__init__.py]: can you test this just to be sure?

Comment (examknow) [reply dronebl/__init__.py]: I have (it works)

Comment (jesopo) [reply dronebl/__init__.py]: awesome. can we change it from `Optional[int]` to `Optional[bool]` and translate `None` (default) in to `2`, `True` in to `1` and `False` in to `0`?

File: dronebl/make.py [hunk 2]
```diff
@@ -10,12 +10,21 @@ def request(
...
+    if listed == True:
+        listedp = "1"
+    elif listed == False:
+        listedp = "0"
+    else:
+        listedp = "2"
+
     element = et.Element("lookup", {
-        "listed": "1"
+        "listed": str(listedp)
```
Comment (jesopo) [inline dronebl/make.py]: final nitpick - given the `listedp` thing fixed the travis warnings, can you remove the `str()` bit?

**Review by jesopo — CHANGES_REQUESTED** [review]

**Comment by jesopo** [comment]
massive thanks

**Pull request merged**